### DISCUSSION TITLE
Please help

### DISCUSSION THREAD

**Discussion by mamadjadali** [payloadcms/payload#5965] [Q&A]
after setting up my local mongodb , after a while my hole db its just gone, it happens 3 time by now , idont know if there is a problem in the code or my db.

**Comment by HarleySalas** [comment]
Can you confirm; your mongodb is **not** accessible over the internet at all?

**Reply by mamadjadali** [reply]
yes it on my local.

**Reply by HarleySalas** [reply]
to help you with this, I think you'd need to share your repo.

The reason I asked about it being accessible over the internet, is because I have had experiences in the past where I put a temporary mongodb instance online, without authentication, or with default user/passwords and within **minutes**, someone gains access, dumps everything and leaves behind a message, trying to ransom the data in exchange for crypto.

**Reply by mamadjadali** [reply]
because it` was on my local in dev mode and offline  i didnt use auth or defualt user/pass, 
i make it public :
https://github.com/mamadjadali/access-control
thank you.

**Reply by HarleySalas** [reply]
I don't see any issues at first glance.

Can you describe what you're doing when the data loss happens? Is it between server restarts? In the middle of developing? It seems to me that you haven't made very many changes from the website template.
